--- mkspecs/win32-g++/qmake.conf.orig   2021-07-22 03:23:03.492109781 +0000
+++ mkspecs/win32-g++/qmake.conf        2021-07-22 03:24:06.345414858 +0000
@@ -53,6 +53,8 @@
 equals(QMAKE_HOST.os, Windows) {
     QMAKE_LINK_OBJECT_MAX = 10
     QMAKE_LINK_OBJECT_SCRIPT = object_script
+} else {
+    QMAKE_LINK_OBJECT_MAX = 1024
 }
 QMAKE_PREFIX_SHLIB      =
 QMAKE_EXTENSION_SHLIB   = dll
--- src/plugins/bearer/platformdefs_win.h.orig  2021-07-22 09:46:38.312164040 +0000
+++ src/plugins/bearer/platformdefs_win.h       2021-07-22 09:47:13.404895527 +0000
@@ -120,6 +120,7 @@

 #endif

+#if 0
 enum NDIS_MEDIUM {
     NdisMedium802_3 = 0,
 };
@@ -129,6 +130,7 @@
     NdisPhysicalMediumBluetooth = 10,
     NdisPhysicalMediumWiMax = 12,
 };
+#endif

 #define OID_GEN_MEDIA_SUPPORTED 0x00010103
 #define OID_GEN_PHYSICAL_MEDIUM 0x00010202
--- src/plugins/bearer/generic/qgenericengine.cpp.orig  2021-07-22 09:44:11.265098452 +0000
+++ src/plugins/bearer/generic/qgenericengine.cpp       2021-07-22 09:44:57.222056637 +0000
@@ -60,6 +60,7 @@
 #  define _WIN32_WINNT 0x0600
 #endif // _WIN32_WINNT < 0x0600
 #include "../platformdefs_win.h"
+#include <ws2ipdef.h>
 #include <iphlpapi.h>
 #endif

--- src/plugins/platforms/windows/qwin10helpers.cpp.orig        2021-07-23 03:56:21.062676305 +0000
+++ src/plugins/platforms/windows/qwin10helpers.cpp     2021-07-23 03:57:04.911599912 +0000
@@ -42,6 +42,7 @@
 #include <QtCore/QDebug>
 #include <QtCore/private/qsystemlibrary_p.h>

+#if 0
 #if defined(Q_CC_MINGW)
 #  define HAS_UI_VIEW_SETTINGS_INTEROP
 // Present from MSVC2015 + SDK 10 onwards
@@ -49,6 +50,7 @@
 #  define HAS_UI_VIEW_SETTINGS_INTEROP
 #  define HAS_UI_VIEW_SETTINGS
 #endif
+#endif

 #include <inspectable.h>

